fdbclient/SystemData.h

```diff
@@ -500,6 +502,7 @@ extern const KeyRangeRef monitorConfKeys;
 extern const KeyRef healthyZoneKey;
 extern const StringRef ignoreSSFailuresZoneString;
 extern const KeyRef rebalanceDDIgnoreKey;
+enum DDIgnore : uint8_t { REBALANCE_DISK = 1, REBALANCE_READ = 2, ALL = 3 };
```

Comment:
enum class is not convenient for bit operations. You have to write `(uint8_t)DDIgnore::REBALANCE_DISK | (uint8_t) DDIgnore::REBALANCE_READ`.
To avoid name conflict, I wrapped the things in a namespace instead.